content/applications/productivity/discuss/overview/get_started.rst

```diff
@@ -1,56 +1,61 @@
+Preferences tab`. Then choose either :guilabel:`Handle by Emails` or :guilabel:`Handle in Odoo` for
+how the user's notifications should be handled.
 
-.. image:: get_started/change_preferences.png
+.. image:: get_started/preferences-user.png
    :align: center
    :alt: View of the preferences page for Odoo Discuss
 
-| By default, the field is set as *Handle by Emails* making messages, notes, and notifications where
-  you were mentioned or that you follow, to be sent through email. By choosing *Handle in Odoo*,
-  they are shown in the *Inbox*.
-| Messages can then be *Marked as Todo*, *Replied*, or *Marked as Read*.
+By default, the field is set as :guilabel:`Handle by Emails`. Every time a message is sent; or a
+note is sent where there is an '@' mention; or when a notification is sent on a record that the
+user's follows, a email message will be sent via the email function in Odoo.
 
-.. image:: get_started/inbox_actions.png
+By choosing :guilabel:`Handle in Odoo`, the above notifications are shown in the *Dicuss* app's
+:guilabel:`Inbox`. Messages can then be :guilabel:`Marked as Todo`, :guilabel:`Replied`,
+:guilabel:`Marked as Read` or responded with an :guilabel:`emoji`.
+
+.. image:: get_started/reactions-discuss.png
    :align: center
    :alt: View of an inbox message and its action options in Odoo Discuss
 
-The messages tagged as *Mark as Todo* are also shown in *Starred*, while the ones *Marked as Read*
-are moved to *History*.
+The messages tagged as :guilabel:`Mark as Todo` are also shown in :guilabel:`Starred`, while the
+ones :guilabel:`Marked as Read` are moved to :guilabel:`History`.
 
 .. image:: get_started/starred_messages.png
    :align: center
    :alt: View of messages marked as todo in Odoo Discuss
 
-Start Chatting
+Start chatting
 ==============
 
-The first time you log in to your account, OdooBot sends you a message asking for permission to
-receive desktop notifications to chats. If accepted, you receive push notifications to the messages
-sent to you despite of where you are in Odoo.
+The first time a user logs in their account, OdooBot sends a message asking for permission to
+receive desktop notifications to chats. If accepted, the user will receive push notifications for
+the messages they receive despite where the user is in Odoo.
 
 .. image:: get_started/odoobot_push.png
    :align: center
    :alt: View of the messages under the messaging menu emphasizing the request for push
          notifications for Odoo Discuss
 
 .. tip::
-   To stop receiving desktop notifications, reset the notifications settings of your browser.
+   To stop receiving desktop notifications, reset the notifications settings of the browser.
 
-To start a chat, click on *New Message* on the *Messaging Menu*, or go to *Discuss* and send a
-*Direct Message*.
+To start a chat go to *Discuss* app and click on the :guilabel:`+` icon next to :guilabel:`Direct
```

Comment:
We're telling the user to select the Discuss app, so it should have a menuselection tag.
```suggestion
To start a chat, go to the :menuselection:`Discuss` app and click on the :guilabel:`+` icon next to :guilabel:`Direct
```